translate/cloud-client/translate_v3_create_glossary.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3_create_glossary]
+from google.cloud import translate_v3 as translate
+
+
+def create_glossary(
+    project_id="[YOUR_PROJECT_ID]",
```

Comment:
nit:
```suggestion
    project_id="YOUR_PROJECT_ID",
```

(same for others)